docs/release-notes/release-notes-0.19.0.md

```diff
@@ -260,6 +260,7 @@ The underlying functionality between those two options remain the same.
     - [Refactor to hide DB transactions](https://github.com/lightningnetwork/lnd/pull/9513)
     - Move the graph cache out of the graph CRUD layer:
       - [1](https://github.com/lightningnetwork/lnd/pull/9533)
+      - [2]()
```

Comment:
nit: could be moved the the next commit.

Reply:
indeed - oversight 👍